Servers/UI/OJS.Servers.Ui/ClientApp/src/components/contests/contest-tasks-navigation/ContestTasksNavigation.tsx

```diff
@@ -70,18 +70,19 @@ const ContestTasksNavigation = () => {
         },
         [ currentProblem, renderIcon, selectProblemById ],
     );
-
+    const sideBartasksList = 'all-tasks-list';
+    const sideBartasksListClass = concatClassNames(styles.tasksListSideNavigation, sideBartasksList);
```

Comment:
By convention, all classnames end with suffix `className`, i.e. this should be `sideBartasksListClassName`